### ISSUE TITLE
[Bug][a11y]: FieldGroup should have role="group" 

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3362]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

FieldGroup

### Expected behavior

FieldGroup should either be implemented using a `fieldset` or have `role="group"` by default.

### Actual behavior

FieldGroup can be labeled using a FieldLabel or `label` attribute, but the rendered `<sp-field-group>` element has no `role`.

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by blunteshwar** [comment]
Done by Westbrook
